kinto/core/storage/postgresql/migrations/migration_017_018.sql

```diff
@@ -0,0 +1,28 @@
+-- Add new deleted column (split into commands is more efficient)
+ALTER TABLE records ADD COLUMN deleted BOOLEAN;
+UPDATE records SET deleted = FALSE;
+ALTER TABLE records ALTER COLUMN deleted SET NOT NULL;
+ALTER TABLE records ALTER COLUMN deleted SET DEFAULT FALSE;
+
+
+-- Lock records and deleted tables before merging them.
+BEGIN WORK;
+LOCK TABLE records IN ACCESS EXCLUSIVE MODE;
+LOCK TABLE deleted IN ACCESS EXCLUSIVE MODE;
+
+INSERT INTO records (id, parent_id, collection_id, data, last_modified, deleted)
+  SELECT id, parent_id, collection_id, '{"deleted": true}'::JSONB, last_modified, TRUE
+  FROM deleted
+-- Because of Bug Kinto/kinto#1375, some tombstones may exist.
+  ON CONFLICT (id, parent_id, collection_id) DO UPDATE
+  SET data = '{"deleted": true}'::JSONB,
+      deleted = TRUE,
+      last_modified = EXCLUDED.last_modified;
+COMMIT WORK;
```

Comment:
This prefers to keep the tombstone. Maybe we should prefer the existing record by running a `DELETE` query first?